slow_tests/test_benchmark.py

```diff
@@ -98,14 +98,6 @@ def benchmark(
         barostat=baro_impl,
     )
 
-    # initialize observables
-    if compute_du_dp_interval > 0:
```

Comment:
`compute_du_dp_interval` can be removed from `benchmark` signature

Reply:
Removed in 3db9cff